autospec/download.py

```diff
@@ -44,10 +44,10 @@ def do_curl(url, dest=None, post=None, is_fatal=False):
         c.setopt(c.POSTFIELDS, post)
     c.setopt(c.FOLLOWLOCATION, True)
     c.setopt(c.FAILONERROR, True)
-    c.setopt(c.CONNECTTIMEOUT, 10)
-    c.setopt(c.TIMEOUT, 10)
+    c.setopt(c.CONNECTTIMEOUT, 30)
```

Comment:
It seems a little much to me to increase this timeout to 30s. From the [docs](https://curl.haxx.se/libcurl/c/CURLOPT_CONNECTTIMEOUT.html) this is the timeout for how long the initial connect takes. Are you experiencing connections that take longer than 10s to establish?

Reply:
@bryteise The current value of 10s for CONNECTTIMEOUT works for my use case. I have proposed a  higher value of 30s for this parameter as its default timeout value is about 300s. 
Will it be fine to have a relaxed timeout for retries if curl request fails due to timeout error.